Library/ViewModels/EmptyCommentsCellViewModel.swift

```diff
@@ -0,0 +1,36 @@
+import KsApi
+import Prelude
+import ReactiveSwift
+
+public protocol EmptyCommentsCellViewModelInputs {
+  /// Call to configure cell with the project
+  func configureWith(project: Project?)
+}
+
+public protocol EmptyCommentsCellViewModelOutputs {
+  /// Emits author's badge for a comment.
+  var emptyText: Signal<String, Never> { get }
```

Comment:
Our output should always match the element name or describe the side-effect so in this case this would be this would be `noCommentsLabelText`. However, the good news is you can remove the view model and its tests here entirely because this value does not change based on the cell configuration, it's always the same (unless it will change later on).

If the value doesn't change then we can be satisfied that it's covered by snapshot tests as part of the UI. If it will change based on configuration then keep the VM.